api/endpoints.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2019 The go-pttai Authors
```

Comment:
so this is basically copied from the previous version of go-ethereum?

Reply:
yes～